test/recur_iterator_test.js

```diff
@@ -675,6 +676,28 @@ suite('recur_iterator', function() {
           '2015-03-31T08:00:00'
         ]
       });
+      testRRULE('FREQ=MONTHLY;INTERVAL=3;BYMONTHDAY=-1', {
```

Comment:
At a glance, it's not clear to me what this test is looking for. A comment would be good, or even a failure reason if `testRRULE()` provides for that.